From 6fb651edcf3c20ead5cc0c73287c698e4f30e942 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Thu, 3 Feb 2022 00:47:59 +0800
Subject: [PATCH 04/15] dt-bindings: edt-ft5x06: add compatible string for
 FT8716 touch func

Focaltech FT8716 is a LCD controller + touchscreen controller combo
chip, with its touch screen functionality accessible via I2C like other
Focaltech touchscreen controllers.

Add a compatible string for it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 .../devicetree/bindings/input/touchscreen/edt-ft5x06.yaml        | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/input/touchscreen/edt-ft5x06.yaml b/Documentation/devicetree/bindings/input/touchscreen/edt-ft5x06.yaml
index 2e8da7470513..5389ecb5fc5e 100644
--- a/Documentation/devicetree/bindings/input/touchscreen/edt-ft5x06.yaml
+++ b/Documentation/devicetree/bindings/input/touchscreen/edt-ft5x06.yaml
@@ -40,6 +40,7 @@ properties:
       - edt,edt-ft5506
       - evervision,ev-ft5726
       - focaltech,ft6236
+      - focaltech,ft8716-ts
 
   reg:
     maxItems: 1
-- 
2.36.0

